<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addialog</class>
 <widget class="QDialog" name="addialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打卡</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>221</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="oLabel">
      <property name="text">
       <string>项目(&amp;O)</string>
      </property>
      <property name="buddy">
       <cstring>proComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="proComboBox">
      <item>
       <property name="text">
        <string>跑步</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>健走</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>跳绳</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>眼保健操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>有氧操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力量训练</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="mLabel">
      <property name="text">
       <string>时长(&amp;M)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tLabel">
      <property name="text">
       <string>日期(&amp;T)</string>
      </property>
      <property name="buddy">
       <cstring>tDateTimeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="tDateTimeEdit">
      <property name="dateTime">
       <datetime>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>8</month>
        <day>24</day>
       </datetime>
      </property>
      <property name="date">
       <date>
        <year>2022</year>
        <month>8</month>
        <day>24</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MonthSection</enum>
      </property>
      <property name="displayFormat">
       <string>MM-dd H:mm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打卡</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
